addons/payment/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,31 @@
+---Fields in module 'payment'---
+payment      / account.invoice          / transaction_ids (many2many)   : NEW relation: payment.transaction
+payment      / payment.acquirer         / qr_code (boolean)             : NEW 
+# NOTHING TO DO
+
+payment      / payment.transaction      / date (datetime)               : NEW
+payment      / payment.transaction      / date_validate (datetime)      : DEL
+# DONE: pre-migration: renamed field
+
+payment      / payment.transaction      / invoice_ids (many2many)       : NEW relation: account.invoice
```

Comment:
This is the twin field of `transaction_ids` in `account.invoice`, so you should group both together.